examples/app_commands/slash_cog_groups.py

```diff
@@ -0,0 +1,44 @@
+from discord.commands.permissions import Permission
+import discord
+from discord.commands.commands import SlashCommandGroup
+from discord.ext import commands
+
+# Set these
+GUILD_ID=0
+OWNER_ID=0
+
+bot = discord.Bot(debug_guild=GUILD_ID, owner_id=OWNER_ID)
```

Comment:
Just a suggestion maybe subclass discord.Bot?

Reply:
Can you please elaborate?